spec/models/service_type_spec.rb

```diff
@@ -14,4 +14,6 @@
     it { should validate_presence_of(:ngsi_version) }
     it { should validate_presence_of(:configuration_template) }
     it { should validate_presence_of(:deploy_template) }
+    it { should validate_inclusion_of(:is_imported).in_array([true, false]) }
```

Comment:
@taliaga The flag is to mark whether or not a service type is part of the official catalog specified when MasterMind is deployed, or a service type that a user imported themselves, and these are tied to a specific project (so that other projects don't suddenly get a list of imported recipes they don't care about for deploying services)